tests/dragonfly/replication_test.py

```diff
@@ -1085,25 +1085,25 @@ async def test_take_over_counters(df_local_factory, master_threads, replica_thre
                                      port=BASE_PORT,
                                      #  vmodule="journal_slice=2,dflycmd=2,main_service=1",
                                      logtostderr=True)
-    replica = df_local_factory.create(
+    replica1 = df_local_factory.create(
         port=BASE_PORT+1, proactor_threads=replica_threads)
     replica2 = df_local_factory.create(
         port=BASE_PORT+2, proactor_threads=replica_threads)
     replica3 = df_local_factory.create(
         port=BASE_PORT+3, proactor_threads=replica_threads)
-    df_local_factory.start_all([master, replica, replica2, replica3])
+    df_local_factory.start_all([master, replica1, replica2, replica3])
     async with (
         master.client() as c_master,
-        replica.client() as c_replica,
+        replica1.client() as c1,
         master.client() as c_blocking,
         replica2.client() as c2,
         replica3.client() as c3,
     ):
-        await c_replica.execute_command(f"REPLICAOF localhost {master.port}")
+        await c1.execute_command(f"REPLICAOF localhost {master.port}")
         await c2.execute_command(f"REPLICAOF localhost {master.port}")
-        await c3.execute_command(f"REPLICAOF localhost {replica.port}")
+        await c3.execute_command(f"REPLICAOF localhost {replica1.port}")
```

Comment:
you are creating replica of a replica..
This is not a feature that we claim to support, it might be working for you in the test, but it is actually no a valid configuration that we expect from user. We probably need to block this configuraiton it in our code

Reply:
done